packages/neovim/src/utils/findNvim.ts

```diff
@@ -117,35 +117,65 @@ function compareVersions(a: string, b: string): number {
   return 0;
 }
 
+function getPlatformPaths() {
+  const paths = new Set<string>();
+  const { PATH, USERPROFILE, LOCALAPPDATA, PROGRAMFILES, HOME } = process.env;
+
+  const normalizePath = (path: string) => normalize(path.toLowerCase());
+
+  PATH?.split(delimiter).forEach(p => paths.add(normalizePath(p)));
+
+  // Add common Neovim installation paths not always in the system's PATH.
+  if (windows) {
+    // Scoop common install location
+    if (USERPROFILE) {
+      paths.add(normalizePath(`${USERPROFILE}\\scoop\\shims`));
+    }
+    paths.add(normalizePath('C:\\ProgramData\\scoop\\shims'));
+
+    // Winget common install location
+    if (LOCALAPPDATA) {
+      paths.add(normalizePath(`${LOCALAPPDATA}\\Microsoft\\WindowsApps`));
```

Comment:
I'm not sure this is right default install location for winget. I tested it on my window virtual machine and it turns out neovim was installed in `C:ProgramFiles\Neovim\bin`.

I also found [this](https://github.com/microsoft/winget-cli/blob/master/doc/specs/%23182%20-%20Support%20for%20installation%20of%20portable%20standalone%20apps.md) from their docs.

Reply:
It's worth a try, thanks for researching that! Would be good to mention that URL in the comment too.